src/applications/facility-locator/components/search-results/LocationDirectionsLink.jsx

```diff
@@ -5,15 +5,24 @@ import { buildAddressArray } from '../../utils/facilityAddress';
 class LocationDirectionsLink extends Component {
   render() {
     const { location } = this.props;
-    const address = buildAddressArray(location).join(', ');
+    let address = buildAddressArray(location);
+
+    if (address.length !== 0) {
+      address = address.join(', ');
+    } else {
+      // If we don't have an address fallback on coords
+      const { lat, long } = location.attributes;
+      address = `${lat},${long}`;
+    }
 
     return (
       <span>
         <a
```

Comment:
Not directly related to your changes: can we add `rel="noopener"` here?

Reply:
Good call. I'd been adding that to other anchors across other portions of the Locator, too.